chapters/101/index.md

```diff
@@ -1,167 +1,105 @@
 
 
-<span style="display: inline-block; width: 60%;">
+### Segmenter
 
-![Jina 101 Family of Executor, Copyright by Jina AI Limited](./img/ILLUS4.png)
+![](images/segmenter.svg)
 
-</span>
+Like a [Crafter](images/#crafter), a Segmenter also pre-processes Documents. A Segmenter breaks Documents into **multiple** chunks. For example, breaking a paragraph into sentences.
 
 
-**Executor is a big family.** Each family member focuses on one important aspect of the search system. Let’s meet:
-- **Crafter**: for crafting/segmenting/transforming the documents and chunks
-- **Encoder**: for representing the chunks as vectors
-- **Indexer**: for saving and retrieving vectors and key-value information from storage
-- **Ranker**: for sorting the results
+### Encoder 
 
-Having a new algorithm in mind? No problem, this family always welcome new members!
+![](images/encoder.svg)
 
+Encoders create vector embeddings of Documents, letting users perform semantic search. 
 
-## Driver
-
-<span style="display: inline-block; width: 30%;">
 
-![Jina 101 Driver, Copyright by Jina AI Limited](./img/ILLUS5.png)
+### Indexer
 
-</span>
+![](images/indexer.svg)
 
+After Documents are [encoded](images/#encoder), an Indexer:
 
-**Driver defines how an Executor behaves on network requests.** Driver helps the Executor to handle the network traffic by interpreting the traffic data (e.g. Protobuf) into the format that the Executor can understand and handle (e.g. Numpy array).
+*   **Saves** Documents’ vector embeddings and metadata key-values pairs to storage (during indexing).
+*   **Retrieves** the vector embeddings and key-value pairs from storage (during querying).
 
 
+### Ranker
 
+![](images/ranker.svg)
 
-## Pea
+Rankers sort and rank search results. They are frequently used in a querying Flow. 
 
-<span style="display: inline-block; width: 30%;">
 
-![Jina 101 Pea, Copyright by Jina AI Limited](./img/ILLUS6.png)
+### Classifier
 
-</span>
+![](images/classifier.svg)
 
-<span style="display: inline-block; width: 30%; float:right" >
+Classifiers sort input Documents into categories and output the predicted hard/soft labels. Classifiers are optional, but may be useful depending on the use case.
 
-![Jina 101 Container Pea, Copyright by Jina AI Limited](./img/ILLUS7.png)
 
-</span>
+### Evaluator
 
-**Pea wraps an Executor and grants it the ability to exchange data over a network.** Pea can send and receive data from other Peas. Pea can also run inside a Docker container, containing all dependencies and the contextual environment in one place.
+![](images/evaluator.svg)
 
+Not every search gets perfect results the first time. An Evaluator evaluates performance of other Executors by comparing the Document rankings you **get** with the Document rankings you **want**. Choosing the right set of Executors optimizes your search.
```

Comment:
@JoanFM How about 

```
Not every search gets perfect results the first time. An Evaluator evaluates the performance of other Executors, for example, by comparing the Document rankings you **get** with the Document rankings you **want**. 
```
@alexcg1 
Since you wrote the part for Evaluator. 
Regarding `Choosing the right set of Executors optimizes your search. `
Is it supposed to mean the right set of `Evaluators`?

Reply:
I would say it evaluates the performance of a system

Reply:
@JoanFM do you mean changing this part like
```
Evaluators evaluate the performance of your search system, for example, by comparing the Document rankings you **get** with the Document rankings you **want**.
```


The original explanation `An Evaluator evaluates performance of other Executors` are probably based on this in our docs:

![Screenshot from 2021-02-06 15-45-50](https://user-images.githubusercontent.com/48913707/107121445-c84dfc00-6892-11eb-8ae7-a60918f47761.png)

Reply:
No do not worry I think my solution may be more confusing actually

Reply:
Specially for 101